### ISSUE TITLE
About the training？

### ISSUE THREAD

**Issue by pupu-chenyanyan** [GigaAI-research/ReconDreamer#1]
Hello, I wonder what level of GPU  and days are required to train this model and  days？Can you provide training commands and related configuration files in the README ?

Thank you for your wonderful work！

**Comment by GigaAI-research** [comment]
Yes, we will release more code after it is accepted by a conference or journal.

**Comment by LivingTom** [comment]
+1

**Comment by manggiechen** [comment]
> Yes, we will release more code after it is accepted by a conference or journal.

Congratulations on your work being accepted by CVPR. When will the training code be released？

**Comment by LivingTom** [comment]
> Hello, I wonder what level of GPU and days are required to train this model and days？Can you provide training commands and related configuration files in the README ?
> 
> Thank you for your wonderful work！



> Yes, we will release more code after it is accepted by a conference or journal.
hi , clould you have any plane to release the training code ?

**Comment by hakuturu583** [comment]
Congraturations!
I am looking forward to train this model!

**Comment by shaopeng666** [comment]
Congraturations!
I am looking forward to train this model!
